src/kotlin/.gitignore

```diff
@@ -0,0 +1,15 @@
+*.iml
+.gradle
+/local.properties
+/.idea/caches
+/.idea/libraries
+/.idea/modules.xml
+/.idea/workspace.xml
+/.idea/navEditor.xml
+/.idea/assetWizardSettings.xml
+.DS_Store
+/build
+/captures
+.externalNativeBuild
+.cxx
+local.properties
```

Comment:
I am wondering if we should move this to the top-level .gitignore file. Wdyt?

Reply:
It's good idea, I'll move it to top level